External/FEXCore/Source/Interface/Core/LookupCache.h

```diff
@@ -78,7 +84,8 @@ class LookupCache {
     // Do L1
     auto &L1Entry = reinterpret_cast<LookupCacheEntry*>(L1Pointer)[Address & L1_ENTRIES_MASK];
     if (L1Entry.GuestCode == Address) {
-      L1Entry.GuestCode = L1Entry.HostCode = 0;
+      L1Entry.GuestCode = 0;
+      // leave L1Entry.HostCode as is
```

Comment:
It is there to make it clear that not clearing it is intentional, and not a bug or something that was forgotten.

From the PR description
```LookupCache/L1 is "soft invalidated" by changing the guest address to zero. Depending on coherency, other threads might still read the old value, however as the lookup value isn't written this should not tear. The write lock guarantees write-write coherency```

That is a soft guarantee, as this PR isn't meant/tested to actually handle invalidations while doing lookups.

Reply:
> It is there to make it clear that not clearing it is intentional, and not a bug or something that was forgotten.

Okay, so... *why* are we not clearing it? 😛
Is it because other threads might still read the old value and hence access the HostCode? The PR explanation is good but people aren't going to find it when looking at this particular line. How about factoring the former into this comment?

Reply:
(the comment has been extended)